local-perf/terra-helm-local/charts/workspacemanager-performance/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
```

Comment:
Having these charts in local-perf/terra-helm-local/* kind of confuses me. The setup_local_env scripts clones in the terra-helm and -helmfile repos, but it looks like you're actually using local charts instead of the ones from the cloned repo.

I think you probably want put the charts in the helm repo (https://github.com/broadinstitute/terra-helm) and reference the locally cloned repo

Reply:
Correct! The reason we are using `terra-helm-local` here is because we haven't yet put the performance test chart in `terra-helm` repo. This is pending further sync up with DevOps in the coming QA / DevOps sync up.